src/oop/object_independence/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-object_independence-medium-q2
+
+## Question Info
+
+- Question type: Parsons
+
+## Question Text
+
+You've noticed that there are line-ups everywhere. So, you decided to write a program that predicts how long you need to
+wait in them. Make a program called  **PhoneWaitList** that has an attribute called **umAhead**, with a basic accessor
+and mutator method and a **toString()** method.
+
+### PhoneWaitList Class
+
+Your `PhoneWaitList` class must have the following items:
+
+- Attributes:
+    - An `int` called **numAhead**
+    - Note that neither of these attributes should be visible from another class
+
+- Constructors:
+    - A single constructor with a parameter `int` that initalzies the **numAhead** attribute
+    - This constructor should be visible from another class
+
+- Methods:
+    - a `getNumAhead` method which is accessible from other classes and returns the value of the **numAhead** attribute
+    - a `setNumAhead` method which accepts an `int` value, updates the **numAhead** attribute with it, and
+      is accessible from other classes
+    - a `toString` method which returns a String of the information about the `PhoneWaitList` class of the form
+      `There are " + numAhead + " people ahead of you`, and is visible from other classes.
```

Comment:
weird that in the other two method descriptions it says accessible from other classes but here it says visible